spec/index.html

```diff
@@ -246,7 +254,8 @@ <h2>Notation and Terminology</h2>
       so that each triple in a graph is considered to be a subgraph.
       A <dfn>proper subgraph</dfn> is a proper subset of the triples in the graph.</p>
 
-    <p>A <dfn>ground</dfn> RDF graph is one that contains no blank nodes.</p>
+    A <dfn>ground</dfn> RDF term is one that contains no blank nodes.</p>
+
```

Comment:
```suggestion
```
Redundant: ground RDF term is now already defined in line 199.

Reply:
Correct